app/src/components/generic/recordMenuCard/RecordMenuCard.tsx

```diff
@@ -15,33 +17,38 @@ type SubSectionProps = {
     setStage: Dispatch<SetStateAction<LG_RECORD_STAGE>>;
 };
 
-function RecordMenuCard({ recordLinks, setStage }: Props) {
+function RecordMenuCard({ recordLinks, setStage, showMenu, className }: Props) {
     const updateActions = recordLinks.filter((link) => link.type === RECORD_ACTION.UPDATE);
```

Comment:
Add the default classname here in the props deconstruction props = 'classnamestring'. This means it still has a prop it can pass but if nothing is passed itll use this default string. It prevents needing to type it in the test cases.